website/docs/networking/vpc-lattice/latticeabtesting.md

```diff
@@ -0,0 +1,115 @@
+# Set up Lattice Service Network
+
+1. Create the Kubernetes Gateway `eks-workshop-gw`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/eks-workshop-gw.yaml
+   ```
+1. Verify that `eks-workshop-gw` gateway is created:
+   ```bash
+   $ kubectl get gateway  
+   ```
+   ```
+   $ NAME              CLASS         ADDRESS   READY   AGE
+   $ eks-workshop-gw   aws-lattice                     12min
+   ```
+
+1. Once the gateway is created, find the VPC Lattice service network. Wait until the status is `Reconciled` (this could take about five minutes).
+   ```bash
+   $ kubectl describe gateway eks-workshop-gw
+   ```
+   ```
+   $ apiVersion: gateway.networking.k8s.io/v1alpha2
+   kind: Gateway
+   ...
+   $ status:
+   conditions:
+   message: 'aws-gateway-arn: arn:aws:vpc-lattice:us-west-2:<YOUR_ACCOUNT>:servicenetwork/sn-03015ffef38fdc005'
+   reason: Reconciled
+   status: "True"
+   ```
+ Now you can see the associated Service Network created in the VPC console under the Lattice resources.
+![Checkout Service Network](assets/servicenetwork.png)
+
+# Create Routes to targets
+We will show how to preform A/B testing between the two version (`checkout`and `checkoutv2`) using `HTTPRoutes`.
+
+**NOTE:** (At the time of writing, the `targetPort` has to be updated  to `8080` for the Lattice Controller to reroute the traffic). This will be improved in future releases.
+
+```bash
+$ kubectl patch svc checkout -n checkout --patch '{"spec": { "type": "ClusterIP", "ports": [ { "name": "http", "port": 80, "protocol": "TCP", "targetPort": 8080 } ] } }'
+$ kubectl patch svc checkout -n checkoutv2 --patch '{"spec": { "type": "ClusterIP", "ports": [ { "name": "http", "port": 80, "protocol": "TCP", "targetPort": 8080 } ] } }' 
+```
+
+1. Create the Kubernetes `HTTPRoute` route that evenly distributes the traffic between `checkout` and `checkoutv2`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/routes/checkout-route.yaml
+   ```
+   ```file
+   /networking/vpc-lattice/routes/checkout-route.yaml
+   ```
+
+2. Find out the `HTTPRoute`'s DNS name from `HTTPRoute` status (highlighted here on the `message` line):
+
+```bash
+$ kubectl describe httproute checkoutroute -n checkout
+```
+
+```bash
+$ apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+    ...
+status:
+parents:
+- conditions:
+    - lastTransitionTime: "2023-02-27T14:36:26Z"
+    message: 'DNS Name: checkout-checkouroute-05bcd5fb087c79394.7d67968.vpc-lattice-svcs.us-west-2.on.aws'
+    reason: Reconciled
+    status: "True"
+    type: httproute
+    controllerName: application-networking.k8s.aws/gateway-api-controller
+    parentRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: eks-workshop-gw
+```
+
+ Now you can see the associated Service created in the VPC console under the Lattice resources.
+![CheckoutRoute Service](assets/checkoutroute.png)
+
+3. Patch the configmap to point to the new endpoint.
```

Comment:
This is the key learning of the whole section. Can we add more details about what is happening under the covers? Without additional details, this does not impart any learning. Adding links to docs would be sufficient too.

Reply:
Agree, added a `tip` section to highlight this part.